onnx/defs/tensor/defs.cc

```diff
@@ -3720,4 +3720,65 @@ ONNX_OPERATOR_SET_SCHEMA(
           return true;
         }));
 
+bool BuildContextDependentFunctionBodyAttributeHasValue(
+    const FunctionBodyBuildContext& ctx,
+    const OpSchema& schema,
+    FunctionProto& functionProto) {
+  FunctionBuilder builder(functionProto);
+  auto ints_attr_proto = ctx.getAttribute("ints");
+  bool has_ints = (nullptr != ints_attr_proto) && ints_attr_proto->ints_size() > 0;
+  auto int_attr_proto = ctx.getAttribute("int");
+  bool has_int = (nullptr != int_attr_proto) && int_attr_proto->has_i();
```

Comment:
I guess you will add a check for all other attributes as well?

Reply:
May be something like:
```cpp
   for (const char *name : { "int", "ints", ... })
      if ctx.getAttribute(name) != nullptr {
         has_attr = true;
         break;
      }
```

Reply:
Yes I will find a way to iterate all types defined in proto. For now, I just want to covert ints so that I can validate it in converting reduce ops to function ops.